### PR TITLE
Attractive and informative front page.

### PR DESCRIPTION
**Pull request by Adaickalavan**
Edited the front page to become more attractive and informative.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -85,7 +85,8 @@
             "pytest-xdist>=2.4.0",
         ],
         "train": [
-            "opencv-contrib-python-headless==4.1.2.30",
+            "opencv-python==4.1.2.30",
```
Comment (liu-allan) [inline setup.py:88]: is this necessary?

File: smarts/env/__init__.py [hunk 2]
```diff
@@ -17,9 +17,27 @@
...
     entry_point="smarts.env.hiway_env:HiWayEnv",
 )
+
+register(
+    id="intersection-v0",
+    entry_point="smarts.env.intersection_env:intersection_env",
+)
+
+
+def build_scenario(scenario: List[str]):
```
Comment (liu-allan) [inline smarts/env/__init__.py:36]: nitpick but should `build_scenario` belong somewhere in `smarts/sstudio`? seems more related there

File: setup.py (hunk 1 above)
Comment (Adaickalavan) [reply setup.py:88]: Unfortunately, installing `ray[rllib]==1.0.1.post1` appears to end up installing both `opencv-python` and `opencv-python-headless`. The opencv version of `4.1.2.30` was explicitly specified as the later versions seem to be incompatible with `ray[rllib]==1.0.1.post1`.

File: smarts/env/__init__.py (hunk 2 above)
Comment (Adaickalavan) [reply smarts/env/__init__.py]: Moved.

File: README.md [hunk 3]
```diff
@@ -204,78 +225,69 @@ env = gym.make(
...
 )
 ```
-
-### Interfacing w/ PyMARL and malib
-
-[PyMARL](https://github.com/oxwhirl/pymarl) and [malib](https://github.com/ying-wen/malib) have been open-sourced. You can run them via,
-
+Below is a sample visualization of an agent's camera sensor observations.
+<p align="center">
+<img src="docs/_static/visdom.gif" ><br/>
```
Comment (Gamenot) [inline README.md:230]: Would you happen to have any idea why the observations are so noisy? It looks like there is some issue with the height, colour estimation, or FoV of the camera that is causing the image to distort.

Comment (Adaickalavan) [reply README.md:230]: Nope.

File: README.md [hunk 4]
```diff
@@ -49,79 +50,123 @@ for _ in range(1000):
...
 cd <path/to/SMARTS>
 
-# Install the system requirements, then follow the instructions for setting up the SUMO_HOME environment variable. You may use the `-y` option to enable automatic assumption of "yes" to all prompts to avoid timeout from waiting for user input. 
+# For Mac OS X users, ensure XQuartz is pre-installed.
+# Install the system requirements, then follow the instructions for setting up the `SUMO_HOME` environment variable. You may use the `-y` option to enable automatic assumption of "yes" to all prompts to avoid timeout from waiting for user input. 
 bash utils/setup/install_deps.sh
 
-# verify sumo is >= 1.5.0
-# if you have issues see ./doc/SUMO_TROUBLESHOOTING.md
+# Verify sumo is >= 1.5.0.
```
Comment (Gamenot) [inline README.md:90]: I think we have been using 1.10.0 recently.

**Review by Gamenot — APPROVED** [review]

**Review by RutvikGupta — APPROVED** [review]
Why does this have changes from PR #1274? We already have a PR for that.

File: README.md (hunk 4 above)
Comment (Adaickalavan) [reply README.md]: This step is now removed with installation of sumo via `pip install eclipse-sumo==1.10.0` .

**Pull request merged**